eyeball/src/lib.rs

```diff
@@ -36,8 +36,9 @@
 //! // If multiple updates have happened without the subscriber being
 //! // polled, the next poll will skip all but the latest.
 //! Observable::set(&mut observable, "C".to_owned());
-//! assert_eq!(subscriber1.next().await, Some("C".to_owned()));
-//! assert_eq!(subscriber2.next().await, Some("C".to_owned()));
+//! Observable::set(&mut observable, "D".to_owned());
+//! assert_eq!(subscriber1.next().await, Some("D".to_owned()));
+//! assert_eq!(subscriber2.next().await, Some("D".to_owned()));
```

Comment:
Hm, both the previous and new code expect there to be two updates? That makes no sense, the second `.await` should block forever... But this is a doctest that actually runs as part of the test suite, right?

Reply:
I was surprised too, but I've tried outside the doctest and it works. I don't know if the doctest is run by the CI.

Reply:
Anyways, can you remove the second assert? We can figure out why it's working later, the example should not rely on it working IMO.

Reply:
Done. Rebased.

Reply:
Oh I see now. Well actually then the original code was correct, it showed that both a previously up-to-date subscriber and a lagged subscriber receive the same value. Not sure if this PR actually makes sense 😅

Reply:
Rebased :-).